lib/krikri.rb

```diff
@@ -12,4 +12,5 @@ module Krikri
   autoload :OaiDcParser,    'krikri/parsers/oai_dc_parser'
   autoload :JsonParser,     'krikri/parsers/json_parser'
   autoload :ModsParser,     'krikri/parsers/mods_parser'
+  autoload :QdcParser,      'krikri/parsers/qdc_parser'
 end
```

Comment:
I believe you have it right.  That line went in with little fanfare in a PR that was parallel to some of the parser work, and it has been frequently overlooked by just about everyone since.  I think we should be able remove all of these autoload lines.